docs/getting-started/install/digitalocean.md

```diff
@@ -1,11 +1,53 @@
-# Digital Ocean
+# Digital Ocean Droplet
 
-On Digital Ocean, there is a pre-made image that can be run for Dokku. You can choose to run this image on any sized droplet, though larger droplets will allow you to run larger applications.
+[Digital Ocean](https://www.digitalocean.com/products/compute/) offers a pre-installed Dokku image. You can run this image on any sized droplet, although larger droplets will allow you to run larger applications.
 
-When choosing your Droplet configuration, please disable IPv6 on the droplet. There are known issues with IPv6 on Digital Ocean and Docker, and many have been reported to the Dokku issue tracker.
+When choosing your Droplet configuration please disable IPv6. There are known issues with IPv6 on Digital Ocean and Docker, and many have been reported to the Dokku issue tracker. If you would like to run Dokku on an IPv6 Digital Ocean Droplet, please consult [this guide](https://jeffloughridge.wordpress.com/2015/01/17/native-ipv6-functionality-in-docker/).
 
-If you would like to run Dokku on an IPv6 Digital Ocean Droplet, please consult [this guide](https://jeffloughridge.wordpress.com/2015/01/17/native-ipv6-functionality-in-docker/) for modifying Docker to run under the Digital Ocean IPv6 configuration.
+## Dokku setup using a Mac
 
-Once Dokku is installed, you can then proceed to the ip address or domain name associated with your server to complete the web-based installation.
+1. Login to your [Digital Ocean](https://m.do.co/c/d716c8c29fb5) account
 
-If you wish for a more unattended installation method, see [these](http://dokku.viewdocs.io/dokku/getting-started/install/debian/#unattended-installation) docs.
+2. Click **Create a Droplet**
+
+3. Under **Choose an image > One-click apps** and choose **Dokku 0.6.5 on 14.04** _(version numbers may vary)_
+
+4. Under **Choose a size** and select your a machine spec
+
+5. Under **Choose a datacenter region** select your region
+
+6. Under **Add your SSH keys** click **New SSH Key** _(this opens a dialog)_
+
+7. From your terminal, execute `cat ~/.ssh/id_rsa.pub`
+
+8. Copy the output and paste it into the **New SSH Key** dialog, provide a name and click **Add SSH Key**
+
+9. Under **Finalize and create**, give your droplet a hostname _(not required)_ and click **Create**
+
+10. Once created, copy the IP address to your clipboard
+
+11. From your terminal, verify you can connect by executing `ssh root@ip-address` - then disconnect using `exit`
```

Comment:
This point until point 16 can and should be performed against the web ui that is started at your server's IP or domain name.